pkg/bitly/main.go

```diff
@@ -0,0 +1,88 @@
+package bitly
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+const SHORTENURL = "https://api-ssl.bitly.com/v4/shorten"
+
+var (
+	// std is the default instance of BitlyAPI.
+	std = New()
+)
+
+// BitlyAPI
+type BitlyAPI struct {
+	token string
+	cli   *http.Client
+}
+
+// New creates a new BitlyAPI with a default HTTP client.
+func New() *BitlyAPI {
```

Comment:
wtf is happening here

Reply:
This function is creating a New instance of the BitlyAPI type and returning a pointer to it.
So when you see `var std = New()`, the `std` variable get's that pointer.

When you include that package, you have access to that pointer through the exported functions.  In this case, the exported functions are `SetToken` and `Shorten(string)`.